service-providers/helixbox/25Q3-collaboration.md

```diff
@@ -0,0 +1,95 @@
+- **Balancer V3 deployment on Darwinia**: The original SnowSwap in our ecosystem has not been updated for a long time, and the DeFi landscape has evolved significantly. We plan to deploy Balancer V3 on the Darwinia chain, providing a more efficient and flexible liquidity solution for users.
+- **KTON Index Fund application development**: An application that allows users to invest in KTON tokens through a diversified index fund, providing a more accessible and user-friendly investment option.
+- **Track and Monitor Applications**: We will continue to monitor the progress of all applications maintained by the RingDAO ecosystem, identifying potential issues and solving them in a timely manner. This includes providing technical support, coordinating development efforts, and ensuring that applications are aligned with the ecosystem's goals.
+
+## Ecosystem Integration and Development
+
+We will continue to drive the integration of third-party systems, there is a tracked list of integration tasks and progress at [ringecosystem/integration](https://github.com/ringecosystem/integration). We will focus on solving these integration tasks and enhancing the ecosystem's interoperability and functionality with the latest developments of the Web3 landscape.
+
+Besides, we will also focus on the following areas:
+
+- **Improve the monitoring system**: Enhance the application server monitoring system to cover key indicators at the business and system levels, including performance, resource usage, error tracking, and alert configuration to ensure stability and rapid incident response.
+- **Daily Maintenance of Blockchain Nodes**: Responsible for the daily maintenance of blockchain nodes, including version upgrades, network connections, block synchronization monitoring, and problem handling to ensure long-term stable operation.
+- **Support CI/CD Pipeline Construction and Optimization**: Support the construction and optimization of CI/CD pipelines, including automated deployment, construction optimization, and troubleshooting, to improve development and operational efficiency.
+
+## Helixbox AI development
+
+One of our key focuses for the upcoming quarter is the development of Helixbox AI, which aims to enhance the user experience and functionality of the Helixbox through AI-driven features. The main components of Helixbox AI include:
+
+- **Helixbox MCP Server**: Integration of LiFi protocol tools for seamless cross-chain swaps and bridges, enabling users to perform cross-chain transactions efficiently.
+- **Helixbox MCP Client**: An intelligent middleware that processes and optimizes MCP server responses, transforming data into frontend-friendly formats and generating components for easier integration.
+- **Helixbox AI Prompts & Intelligence Engine**: A natural language processing engine that enables intuitive user-AI interactions, providing context-aware prompt optimization based on user behavior and preferences.
```

Comment:
Remove this part, Helixbox API is not project of RingDAO.

We can replace with DeGov.AI tasks and scope.